### PR TITLE
Updated Readme to 3.0.0-4

### PR DESCRIPTION
**Pull request by BolverBlitz**

### CODE REVIEW

**Comment by BolverBlitz** [comment]
You also want to update your [Gist](https://gist.github.com/AndrewLaneX/618298c5ef179eebc511ca8c8a82eb76)
```js
const { default: newi18n } = require('new-i18n')
const i18n = new newi18n(__dirname, ["en","pt"], "en");

console.log('English:', i18n.translate('en', 'hello_world'));
console.log('Portuguese:', i18n.translate('pt', 'hello_world'));

console.log(i18n.translate('en', 'hi', { name: '...' }));
console.log(i18n.translate('pt', 'hi', { name: '...' }));

i18n.update('en', { hi: 'Hello my name is {{name}}!' });
i18n.update('pt', { hi: 'Olá meu nome é {{name}}!' });

console.log(i18n.translate('en', 'hi', { name: '...' }));
console.log(i18n.translate('pt', 'hi', { name: '...' }));

console.log(`newi18n languages: ${i18n.languages}`)
```

File: README.md [hunk 1]
```diff
@@ -35,11 +35,22 @@ npm install new-i18n
 `index.js`:
 
 ```js
-const newI18n = require('new-i18n');
-const i18n = newI18n('./localization', ['en', 'pt']);
+const { default: newi18n } = require('new-i18n')
+const i18n = new newi18n(__dirname, ["en","pt"], "en");
```
Comment (vrumger) [inline README.md]:
```suggestion
const { default: I18n } = require('new-i18n');
const i18n = new I18n(__dirname, ['en', 'pt'], 'en');
```

File: README.md [hunk 2]
```diff
@@ -35,11 +35,22 @@ npm install new-i18n
...
+console.log(i18n.translate('en', 'hi', { name: '...' }));
+console.log(i18n.translate('pt', 'hi', { name: '...' }));
+
+i18n.update('en', { hi: 'Hello my name is {{name}}!' });
+i18n.update('pt', { hi: 'Olá meu nome é {{name}}!' });
+
+console.log(i18n.translate('en', 'hi', { name: '...' }));
+console.log(i18n.translate('pt', 'hi', { name: '...' }));
+
+console.log(`newi18n languages: ${i18n.languages}`)
```
Comment (vrumger) [inline README.md]:
```suggestion
console.log(`I18n languages: ${i18n.languages}`);
```

**Review by vrumger — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -55,16 +66,16 @@ i18n('pt', 'hello_world'); // 'Olá Mundo'
...
 
 # Updating Varaibles
 
 [//]: # '{% raw %}'
 
 ```js
-i18n.update('en', { hi: 'Hello {{name}}!' });
-i18n('en', 'hi', { name: '...' }); // 'Hello ...!'
+i18n.update('en', { hi: 'Hello my name is {{name}}!' });
+i18n.translate('en', 'hi', { name: '...' }); // 'Hello ...!'
```
Comment (vrumger) [inline README.md]:
```suggestion
i18n.translate('en', 'hi', { name: '...' }); // 'Hello my name is ...!'
```

**Review by vrumger — CHANGES_REQUESTED** [review]

**Comment by vrumger** [comment]
Sorry about the delay.

**Pull request merged**